docs/source/release_notes/release-notes.rst

```diff
@@ -50,6 +50,14 @@ develop
     *    - Type
          - Change
 
+    *    - |userbreak|
+         - The ``timelockRuntime`` block is now mandatory. Before this it was optional.
```

Comment:
The two userbreaks can be collapsed into one IMHO. It would be good to link to the docs here too.